cmd/common.go

```diff
@@ -77,6 +77,13 @@ func getConfig(cmd *cobra.Command) types.PlatformConfig {
 		log.Infof("Running a dry-run mode, no changes will be made")
 	}
 
+	patchFilePath, _ := cmd.Flags().GetString("path")
+	base.PatchPath = patchFilePath
```

Comment:
you assign `patchFilePath` to `base.PatchPath` and then check `patchFilePath` if it is not empty. Could you explain why?

Reply:
no specific reason, can assign to base.PatchPath directly. changes done.